docs/user/reference/secret.md

```diff
@@ -120,6 +120,23 @@ The `vault` backend is the only one that supports specific configuration keys. T
 
 A minimum configuration must include the `endpoint` and `token`. However, just that would not be insecure, as it wouldn't establish an encrypted TLS connection to Vault. For production you should configure your Vault securely, following recommendations in the upstream Vault documentation.
 
+The `kubernetes` backend supports the following configuration keys:
+
+|||
+|---|---|
+| `ca-cert` | A PEM-encoded CA certificate. This comes from base64 decoding the `certificate-authority-data` value in the kube config file.|
+| `ca-certs` | A list of A PEM-encoded CA certificates (if a cert chain is used).|
+| `client-cert` | A PEM-encoded client certificate. This comes from base64 decoding the user's `client-certificate-data` value in the kube config file.|
+| `client-key` | An unencrypted, PEM-encoded client private key. This comes from base64 decoding the user's `client-key-data` value in the kube config file.|
+| `endpoint` ||
+| `namespace` | The namespace to use store the secrets. The namespace must already exist (it is not created).|
+| `skip-tls-verify` | Do not verify the TLS certificate. For testing only.|
+| `token` | The k8s authentication token (can be generated using `kubectl create token default --namespace ${namespace}`.|
+| `username` | The k8s authentication username.|
+| `password` | The k8s authentication password.|
+
+A minimum configuration must include the `endpoint`, `namespace`, and `ca-cert`.
+In most cases, `token` would also be specified for authentication.
 
```

Comment:
In this whole table, I'd 

- replace "kube config" with "kubeconfig" (the more common spelling).
- replace "k8s" with "K8s" or "Kubernetes" (the more formal spelling).

Reply:
ok